internal/commands/util.go

```diff
@@ -39,7 +38,6 @@ func help(ctx context.Context, app *app.App, channelID string) {
 		app.Logger.Error(
 			ctx,
 			"command/util.help postMessage error",
-			log.NewValue("channelID", channelID),
```

Comment:
Why removing the channelID field from this log but not adding a logWriter with it to use instead of app.Logger?